FileIO/VtkIO/PVDFile.cpp

```diff
@@ -0,0 +1,46 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include <fstream>
+#include <iomanip>
+#include <limits>
+#include <logog/include/logog.hpp>
+
+#include "PVDFile.h"
+
+namespace FileIO
+{
+
+PVDFile::PVDFile(const std::string &fn)
+    : _fh{fn.c_str()}
+{
+    if (!_fh) {
+        ERR("could not open file `%s'", fn.c_str());
+        std::abort();
+    }
+
+    _fh << std::setprecision(std::numeric_limits<double>::digits10);
+
+    _fh << "<?xml version=\"1.0\"?>\n"
+           "<VTKFile type=\"Collection\" version=\"0.1\" byte_order=\"LittleEndian\""
+           " compressor=\"vtkZLibDataCompressor\">\n"
+           "  <Collection>\n";
+}
+
+void PVDFile::addVTUFile(const std::string &fn, double timestep)
+{
+    _fh << "    <DataSet timestep=\"" << timestep << "\" group=\"\" part=\"0\" file=\"" << fn << "\"/>\n";
+}
+
+PVDFile::~PVDFile()
+{
+    _fh << "  </Collection>\n</VTKFile>\n";
```

Comment:
From my point of view, the XML data is not resource, rather outcome of this functionality. 

I think the interface is not intuitive because usually we (maybe only me?) do not expect a destructor doing something beyond resource finalization (completing the XML data is not included here). Besides, one needs to write the following codes to make it sure the XML is complete at some point. 

```
{
PVDFile pvd("test.pvd");
pvd.addVTUFile();
}

// do something using the PVD file
```

The following codes looks more readable 
```
PVDFile pvd("test.pvd");
pvd.addVTUFile();

// do something using the PVD file
```

or 

```
PVDFile pvd("test.pvd");
pvd.addVTUFile();
pvd.close();

// do something using the PVD file
```